resources/js/src/app/components/customer/login/ForgotPassword.js

```diff
@@ -24,9 +24,9 @@ Vue.component("forgot-password-modal", {
     data()
     {
         return {
-            password: "",
```

Comment:
Wird password nicht mehr benötigt?

Reply:
Password wurde in der Komponente nicht verwendet, und im Template ebenfalls nicht.
Von daher hab ich es entfernt.